mdpow/tests/test_equilibration_script.py

```diff
@@ -0,0 +1,39 @@
+import tempdir as td
+import os
+import manifest
+
+from mdpow.run import equilibrium_simulation
+from mdpow.config import get_configuration
+
+class TestEquilibriumScript(object):
+    
+    def setup(self):
+        self.tmpdir = td.TempDir()
+        self.old_path = os.getcwd()
+        self.resources = self.old_path + "/mdpow/tests/testing_resources"
+        m = manifest.Manifest(os.path.join(self.resources,'manifest.yml'))
+        m.assemble('base',self.tmpdir.name)
+
+    def _run_equil(self, solvent, dirname):
+        try:
+            print(os.listdir('.'))
+            cfg = get_configuration('runinput.yml')
+            self.S = equilibrium_simulation(cfg, solvent, dirname=dirname)
+        except:
+            assert False
+
+    def test_basic_run(self):
+        os.chdir(self.tmpdir.name)
```

Comment:
Or we should at least create our own context manager, eg [gromacs.utilties.in_dir](https://github.com/Becksteinlab/GromacsWrapper/blob/develop/gromacs/utilities.py#L256):
```python
with in_dir(self.tmpdir.name, create=False):
   do_stuff
```
and in won't destroy the directory.

Either import from GW or copy the definition.